modules/creating-serverless-apps-kn.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// serverless/serving-creating-managing-apps.adoc
+
+[id="creating-serverless-apps-kn_{context}"]
+= Creating serverless applications using the kn CLI
+
+.Prerequisites
+* Installed the {ServerlessOperatorName} and Knative Serving.
+* Installed `kn` CLI.
+
+.Procedure
+
+. Create the Knative service by entering:
++
+----
+$ kn service create <SERVICE-NAME> --image <IMAGE-URL> --env <ENVIRONMENT-VARIABLE-NAME=VALUE>
+----
++
+.Example
++
+----
+kn service create hello --image gcr.io/knative-samples/helloworld-go --env TARGET=Knative
+
+Creating service 'hello' in namespace 'default':
+
+  0.247s The Route is still working to reflect the latest desired specification.
+  0.299s Configuration "hello" is waiting for a Revision to become ready.
+ 11.631s ...
+ 11.719s Ingress has not yet been reconciled.
+ 13.102s Ready to serve.
```

Comment:
surely we can do faster than that! ;)

After a few tries, totally reprezentative sample

```
  0.271s The Route is still working to reflect the latest desired specification.
  0.580s Configuration "hello" is waiting for a Revision to become ready.
  3.857s ...
  3.861s Ingress has not yet been reconciled.
  4.270s Ready to serve.
```